entrypoint.sh

```diff
@@ -71,6 +71,14 @@ git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
 git remote rm origin || true
 git remote add origin "${remote_repo}"
 git add --all
-git commit --allow-empty -m "Automated deployment: $(date -u) ${GITHUB_SHA}"
+
+print_info "Allowing empty commits: ${INPUT_EMPTYCOMMITS}"
+COMMIT_MESSAGE="Automated deployment: $(date -u) ${GITHUB_SHA}"
+if [[ ${INPUT_EMPTYCOMMITS} == "true" ]]; then
+    git commit --allow-empty -m "${COMMIT_MESSAGE}"
+else
+    git commit -m "${COMMIT_MESSAGE}" || true
```

Comment:
How about the following change to get information?

```diff
- git commit -m "${COMMIT_MESSAGE}" || true
+ git commit -m "${COMMIT_MESSAGE}" || \
+     print_info "nothing to commit, working tree clean" && \
+     true
```

Reply:
That's probably a better option than I had. I'll make the change, but maybe augment the message to something a little more descriptive, like:

> No changes detected, skipping deployment

Reply:
Good! I will fix it.

Reply:
Thanks